app/ui-angular/pom.xml

```diff
@@ -205,29 +216,29 @@
             <executions>
               <execution>
                 <id>set https proxy</id>
-                <phase>initialize</phase>
+                <phase>generate-sources</phase>
                 <goals>
-                  <goal>yarn</goal>
+                  <goal>npm</goal>
                 </goals>
                 <configuration>
                   <arguments>config set https-proxy http://${proxy-user}:${proxy-password}@${proxy-server}:${proxy-port}</arguments>
                 </configuration>
               </execution>
               <execution>
                 <id>set http proxy</id>
-                <phase>initialize</phase>
+                <phase>generate-sources</phase>
                 <goals>
-                  <goal>yarn</goal>
+                  <goal>npm</goal>
```

Comment:
I'm curious as to why this needs to be npm instead of yarn as we use yarn exclusively.

Reply:
Changed, I'm not sure it solely with yarn, but once this goes in I can test it out on staging.